package/libcurl/package

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+# Copyright (c) 2021 The Toltec Contributors
+# SPDX-License-Identifier: MIT
+
+pkgnames=(libcurl-dev)
+pkgdesc="Client-side URL transfer library"
+url=https://curl.se/libcurl
+pkgver=7.75.0-1
+timestamp=2021-04-14T00:00:00Z
+section="devel"
+maintainer="TBD"
+license=MIT
+
+image=base:v1.5
+source=("https://curl.se/download/curl-${pkgver%-*}.tar.gz")
+sha256sums=(4d51346fe621624c3e4b9f86a8fd6f122a143820e17889f59c18f245d2d8e7a6)
+
+build() {
+    ./configure --prefix=/opt --host="$CHOST"
+    make
+    DESTDIR="$srcdir"/install make install
+}
+
+package() {
+    install -D -m 644 -t "$pkgdir"/opt/lib/ \
+        "$srcdir"/install/opt/lib/libcurl.a "$srcdir"/install/opt/lib/libcurl.la "$srcdir"/install/opt/lib/libcurl.so
```

Comment:
Not sure whether a libcurl.so without any version is supposed to be included in a `-dev`-Package. On the other side, there is already a `libcurl` package from entware.

Reply:
After some discussion with Mattéo Delabre on the reMarkable Discord, the binaries should be definilely remove from the `-dev` package. The curl-config is probably also part of the library and should be removed as well.

The final goal should be to make this package a split package (`libcurl` and `libcurl-dev`) where `libcurl` overrides the one from entware to have version parity between both packages. Issue #327 talks about approaches. I personally think that only adding `libcurl-dev` without an accompaning package is ok for now.

Reply:
I oriented myself on [how Debian packages libcurl-dev](https://packages.debian.org/de/jessie/armhf/libcurl4-gnutls-dev/filelist).
As far as I can tell, they follow this structure:
- versioned shared library in the user package
- unversioned shared library, static library, curl-config in the dev package

Reply:
Didn't know that either. I'm not sure what is better indeed. I already discussed it with Matteo, but I think we didn't factor that in. @matteodelabre: Want to have final rule on whether to go the debian route here or remove the binaries?